src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -192,6 +192,27 @@
         hotkeys.toggleCheatSheet();
       }
 
+      function enableWindowResizeMode() {
+        var vm = this;
+        vm.windowResizeModeOn = !vm.windowResizeModeOn;
+        vm.gridsterOpts = {
+          resizable: {
+            enabled: vm.windowResizeModeOn,
+            stop: function (event, $element) {
```

Comment:
I think that if you move the `stop` definition to the original `gridsterOpts` building, this function could be something like:

```js
function enableWindowResizeMode() {
  var vm = this;
  vm.windowResizeModeOn = !vm.windowResizeModeOn;
  vm.gridsterOpts.resizable.enabled = vm.windowResizeModeOn;
  vm.gridsterOpts.draggable.enabled = vm.windowResizeModeOn;
}
```

What do you think?